packages/xo-web/src/xo-app/xosan/new-xosan.js

```diff
@@ -195,10 +248,12 @@ export default class NewXosan extends Component {
     this.setState({ brickSize })
   }
 
-  _selectSr = async (event, sr) => {
-    const selectedSrs = { ...this.state.selectedSrs }
-    selectedSrs[sr.id] = event.target.checked
-    this.setState({ selectedSrs })
+  _selectSrs = selectedSrs => {
+    if (some(selectedSrs, sr => this._getDisableSrCheckbox(sr) === true)) {
```

Comment:
- `_getDisableSrCheckbox` is a selector, not a predicate
- Why `=== true`?
- If you check this before setting the state, the selector will work with an outdated state